package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "webxdao.github.io",
```

Comment:
Hmm, we might want to change the name here. What do you think?

Reply:
It goods repo name goes there